### PR TITLE
Chore/enr 206  eslint config  & prettier to devoinc rules

### PR DESCRIPTION
**Pull request by Bronzevoice**

### CODE REVIEW

**Comment by Worvast** [comment]
@Bronzevoice you include into package-lock.json references to our internal nexus repository, fix it please.

**Review by Worvast — COMMENTED** [review]
Things to fix:

1.- Quit internal nexus repository references
2.- Upgrade version to publish a new one
3.- Update Changelog adding more info of the update, because you add new methods to verify things.

This is a minor not a patch version, remember it.

**Comment by Bronzevoice** [comment]
> Things to fix:
> 
> 1.- Quit internal nexus repository references 2.- Upgrade version to publish a new one 3.- Update Changelog adding more info of the update, because you add new methods to verify things.
> 
> This is a minor not a patch version, remember it.

Ready

**Comment by Worvast** [comment]
LGTM if no one complains in a day or two we merge it

File: lib/client.js [hunk 1]
```diff
@@ -35,7 +57,7 @@ class Client {
   query(options) {
     const opc = this._config.parseQuery(options);
     if (!helper.config.validate(opc)) {
-      return Promise.reject('Invalid options');
+      return Promise.reject(new Error('Invalid options'));
```
Comment (lbermudez) [inline lib/client.js:60]: This is a change that it could be affect to the consumers if they expect an string through reject of the promise.

**Pull request merged**